Sources/CoreAPI/WordPressOrgXMLRPCApiErrorDomain.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+/// Error domain of `NSError` instances that are converted from `WordPressOrgXMLRPCApiError`
+/// and `WordPressAPIError<WordPressOrgXMLRPCApiError>` instances.
+///
+/// See `extension WordPressOrgXMLRPCApiError: CustomNSError` for context.
+let WordPressOrgXMLRPCApiErrorDomain = "WordPressKit.WordPressOrgXMLRPCApiError"
+
+// WordPressOrgXMLRPCApiErrorDomain is accessible only in Swift and, since it's a global, cannot be made @objc.
+// As a workaround, here is a builder to init NSError with that domain and a method to check the domain.
+@objc
+public extension NSError {
+
+    @objc
+    static func wordPressOrgXMLRPCApiError(
+        code: Int,
+        userInfo: [String: Any]?
+    ) -> NSError {
+        NSError(
+            domain: WordPressOrgXMLRPCApiErrorDomain,
+            code: code,
+            userInfo: userInfo
+        )
```

Comment:
If you create a `WordPressOrgXMLRPCApiError` error here, cast it to `NSError`, and return it to Objective-C code, the instance can be cast back to `WordPressOrgXMLRPCApiError` in Swift code when the error is passed from Objective-C code to Swift.

I don't suppose you can cast this `NSError` instance to `WordPressOrgXMLRPCApiError`? It would be pretty magical if you can. 🤔

Reply:
Oh, this is a shortsighted leftover from before I realized that the error domain constant could be defined in `APIInterface` and therefore be available to packages in both Objective-C and Swift.

I shall remove this utils like I did for `WordPressComRestApiErrorDomain`.